src/body_impl.rs

```diff
@@ -122,8 +122,9 @@ fn field_default_expr_and_doc(field: &syn::Field) -> Result<(TokenStream, String
         let field_doc = format!("{}", field_value);
         Ok((field_value, field_doc))
     } else {
+        let field_type = field.ty.clone().into_token_stream();
         Ok((quote! {
-            Default::default()
+            <#field_type as core::default::Default>::default()
```

Comment:
Please use `Default` instead of `core::default::Default`, to avoid breaking projects that use `std`.

Reply:
Sorry for that. I didn't know that using `core` introduces issues for `std`-based projects. Landing fix...

Reply:
Hmmm.... however I'm using this PR now in my projects (all are `std`-based) and no issues happen. Could you explain this point further? When exactly this can be a pain for `std`?